src/meltano/core/project_init_service.py

```diff
@@ -152,8 +154,11 @@ def echo_instructions(self):
         )
 
         click.echo("\nNext steps:")
-        click.secho("  cd ", nl=False)
-        click.secho(self.project_name, fg="magenta")
+
+        if not self.project.root.samefile(os.getcwd()):
+            click.secho("  cd ", nl=False)
+            click.secho(self.project_name, fg="magenta")
+
         click.echo("  Visit ", nl=False)
         click.secho(
             "https://docs.meltano.com/getting-started#create-your-meltano-project",
```

Comment:
```suggestion
            "https://docs.meltano.com/getting-started/part1",
```

While we're here might as well update this!